nilearn/plotting/surface/tests/test_matplotlib_backend.py

```diff
@@ -233,351 +227,3 @@ def test_compute_facecolors_deprecation():
             0.5,
             alpha,
         )
-
-
-def test_plot_surf_with_title(matplotlib_pyplot, in_memory_mesh, bg_map):
-    """Test if figure title is set correctly in
-    nilearn.plotting.surface.surf_plotting.plot_surf.
-    """
-    display = plot_surf(
```

Comment:
I've moved the tests below to `test_surf_plotting` thinking at the end that it is more convenient.